.gitignore

```diff
@@ -13,4 +13,5 @@ cypress/screenshots
 /sass
 /js
 /index.html
-/rivet.zip
\ No newline at end of file
+/rivet.zip
```

Comment:
Can you add a line of blank space here with a comment what this type of file is? so:

```gitignore
/rivet.zip

# Editor files, etc.
*.iml
```